modules/ztp-configuring-disk-partitioning.adoc

```diff
@@ -8,39 +8,179 @@
+----
+--
+
+. Use a tool like link:https://jsonformatter.org/json-pretty-print[JSON Pretty Print] to convert the output into JSON format. 
+
+. Copy the output into the `.spec.clusters.nodes.ignitionConfigOverride` field in the `SiteConfig` CR.
 +
 [source,yaml]
 ----
-nodes:
-    rootDeviceHints:
-      wwn: "0x62cea7f05c98c2002708a0a22ff480ea"
-    diskPartition:
-      - device: /dev/disk/by-id/wwn-0x62cea7f05c98c2002708a0a22ff480ea <1>
-        partitions:
-          - mount_point: /var/imageregistry
-            size: 102500 <2>
-            start: 344844 <3>
+[...]
+spec:
+  clusters:
+    - nodes:
+        - ignitionConfigOverride: |
+          {
+            "ignition": {
+              "version": "3.2.0"
+            },
+            "storage": {
+              "disks": [
+                {
+                  "device": "/dev/disk/by-path/pci-0000:01:00.0-scsi-0:2:0:0",
+                  "partitions": [
+                    {
+                      "label": "var-lib-containers",
+                      "sizeMiB": 0,
+                      "startMiB": 250000
+                    }
+                  ],
+                  "wipeTable": false
+                }
+              ],
+              "filesystems": [
+                {
+                  "device": "/dev/disk/by-partlabel/var-lib-containers",
+                  "format": "xfs",
+                  "mountOptions": [
+                    "defaults",
+                    "prjquota"
+                  ],
+                  "path": "/var/lib/containers",
+                  "wipeFilesystem": true
+                }
+              ]
+            },
+            "systemd": {
+              "units": [
+                {
+                  "contents": "# # Generated by Butane\n[Unit]\nRequires=systemd-fsck@dev-disk-by\\x2dpartlabel-var\\x2dlib\\x2dcontainers.service\nAfter=systemd-fsck@dev-disk-by\\x2dpartlabel-var\\x2dlib\\x2dcontainers.service\n\n[Mount]\nWhere=/var/lib/containers\nWhat=/dev/disk/by-partlabel/var-lib-containers\nType=xfs\nOptions=defaults,prjquota\n\n[Install]\nRequiredBy=local-fs.target",
+                  "enabled": true,
+                  "name": "var-lib-containers.mount"
+                }
+              ]
+            }
+          }   
+[...]
+----
++
+[NOTE]
+====
+Create the field `.spec.clusters.nodes.ignitionConfigOverride` if it does not exist. 
```

Comment:
Provide the reason for doing something before the action. Rewrite as: "If the `.spec.clusters.nodes.ignitionConfigOverride` field does not exist, create it."